qemu-cc/qemu-cc.spec

```diff
@@ -0,0 +1,198 @@
+Name: qemu-cc
+Version: v2.9.0+git.a2e6957b6f
+Release: 0
+Source0: %{name}-%{version}.tar.gz
+Source1: qemu-cc-rpmlintrc
+BuildRoot: %{_tmppath}/%{name}-%{version}-%{release}
+
+Summary  : OpenBIOS development utilities
+Group    : Development/Tools
+License  : BSD-2-Clause BSD-3-Clause GPL-2.0 GPL-2.0+ LGPL-2.0+ LGPL-2.1
+
+Requires: qemu-cc-bin
+Requires: qemu-cc-data
+Requires: qemu-cc-doc
```

Comment:
This isn't necessarily wrong, but why are we now packaging the docs when we didn't in `qemu-lite.spec`?